### DISCUSSION TITLE
[Script request] Wavelog

### DISCUSSION THREAD

**Discussion by DonLocke** [community-scripts/ProxmoxVE#208] [Request script]
### Application Name

Wavelog

### Website

https://github.com/wavelog/wavelog

### Description

[Wavelog](https://www.wavelog.org/) is a self-hosted web-based application for logging amateur radio contacts.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
Wavelog LXC
Date added: 2024-11-12

```bash
bash -c "$(wget -qLO - https://github.com/community-scripts/ProxmoxVE/raw/main/ct/wavelog.sh)"
```
